doc/user/bgp.rst

```diff
@@ -2673,6 +2673,20 @@ the traffic.
 Similarly traffic received from ES peers via the overlay cannot be forwarded
 to the server. This is split-horizon-filtering with local bias.
 
+Knobs for interop
+"""""""""""""""""
+Some vendors do not send EAD-per-EVI routes. To interop with them we
+need to relax the dependency on EAD-per-EVI routes and activate a remote
+ES-PE based on just the EAD-per-ES route.
+
+Note that by default we advertise and expect EAD-per-EVI routes.
+
+.. index:: [no$no] disable-ead-evi-rx
+.. clicmd:: [no$no] disable-ead-evi-rx
```

Comment:
nit: `$no` doesn't need to be here in the command documentation.

Also, it'd be nice to update the bgpd YANG module accordingly to keep it in sync with the CLI.

Reply:
and more importantly, please don't include the "no" in the "index" line - that affects where the command appears in the index.

Reply:
Yeah cut-paste mistakes :) Thx for catching it Renato/Mark, I have fixed up.

And I will follow up on the yang updates needed for the new commands as well.